package.json

```diff
@@ -77,6 +77,7 @@
     "@turf/bbox": "^6.4.0",
     "@turf/centroid": "^6.0.2",
     "canvas-fit": "^1.5.0",
+    "chart2music": "^1.11.0",
```

Comment:
Nice MIT library.
To transpile it to es5, please add it to 
https://github.com/plotly/plotly.js/blob/149c895db2acf3f497a84d62adc276c24d37989b/webpack.config.js#L25

Reply:
Thank you! Knew there'd be something simple I was missing so I could get this into the same style